src/App/App.css

```diff
@@ -25,5 +25,5 @@ p {
   font-size: 20px;
   text-align: center;
   margin-top: 50px;
-  color: whitesmoke;
+  color: black;
```

Comment:
This should be `color: whitesmoke`

Reply:
That'd be the same color as the background and invisible